CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+## 0.5.3
+
+**This release marks the Ruff language server as stable and introduces a new [documentation
```

Comment:
Maybe:

**Ruff 0.5.3 marks the stable release of the Ruff language server and introduces revamped [documentation](https://docs.astral.sh/ruff/editors), including [setup guides for your editor of choice](https://docs.astral.sh/ruff/editors/setup) and [the language server itself](https://docs.astral.sh/ruff/editors/settings)**.

Reply:
0.5.3 seems a bit redundant given the heading, otherwise I like it.